browser/brave_wallet/keyring_service_unittest.cc

```diff
@@ -2129,9 +2292,65 @@ TEST_F(KeyringServiceUnitTest, SetSelectedAccount) {
       hardware_account, "m/44'/60'/1'/0/0", "name 1", "Ledger", "device1",
       mojom::CoinType::ETH));
   service.AddHardwareAccounts(std::move(new_accounts));
-  EXPECT_TRUE(SetSelectedAccount(&service, &observer, hardware_account));
+  EXPECT_TRUE(SetSelectedAccount(&service, &observer, hardware_account,
+                                 mojom::CoinType::ETH));
+  observer.Reset();
+
+  EXPECT_TRUE(Unlock(&service, "brave"));
+  // Can set Filecoin account
+  {
+    absl::optional<std::string> imported_account =
+        ImportFilecoinSECP256K1Account(
+            &service, "Imported Filecoin account 1",
+            // t1gcfbv323mpexk2pumtkgibtvrulxarnafxryyly
+            "8446511b93bb1c63d3ebab9f25aa62939f78dcc062c853000b068bc7468ad134",
+            mojom::kFilecoinTestnet);
+    ASSERT_TRUE(imported_account.has_value());
+    EXPECT_EQ(*imported_account, "t1gcfbv323mpexk2pumtkgibtvrulxarnafxryyly");
+    EXPECT_TRUE(SetSelectedAccount(
+        &service, &observer, imported_account.value(), mojom::CoinType::FIL));
+    observer.Reset();
+    EXPECT_EQ(imported_account.value(),
+              GetSelectedAccount(&service, mojom::CoinType::FIL));
```

Comment:
test `SelectedAccountChanged` for Filecoin

Reply:
done